notebooks/en/semantic_cache_chroma_vector_database.ipynb

```diff
@@ -960,10 +960,17 @@
         "          embedding = self.encoder.encode([question])\n",
         "\n",
         "          # Search for the nearest neighbor in the index\n",
-        "          self.index.nprobe = 8\n",
+        "          self.index.nprobe = 8 # Number of nearby cells to search\n",
+        "\n",
+        "          # D - Array of Distances between the query vector and nearest neighbors\n",
```

Comment:
repetitive comments here breaks readability of the code sometimes, would be nice to instead have a docstring